provider/terraform/tests/resource_dns_managed_zone_test.go

```diff
@@ -6,7 +6,6 @@ import (
 
 	"github.com/hashicorp/terraform/helper/acctest"
 	"github.com/hashicorp/terraform/helper/resource"
-	"github.com/hashicorp/terraform/terraform"
 )
 
 func TestAccDnsManagedZone_basic(t *testing.T) {
```

Comment:
This basic test can probably also be removed since it's essentially identical to the generated one

Reply:
I decided to do the other way around, leave the original tests and remove all the generated ones as they didn't look right.

Reply:
What was wrong with them? If our test generator is doing something wrong, I'd rather it get fixed than have a bunch of special casing around.

Reply:
I've readded the autogen tests with the random_id discussed in the other thread. It's now passing the acceptance tests for Terraform.